.travis.yml

```diff
@@ -4,21 +4,28 @@ addons:
 services:
   - postgresql
 
-before_script:
-  - psql -c 'create role simplified_test;' -U postgres
-  - psql -c 'create database simplified_content_test;' -U postgres
-  - psql -c 'grant all privileges on database simplified_content_test to simplified_test;' -U postgres
-
 language: python
 
 python:
   - "2.7"
 
+cache: pip
+
+git:
+  submodules: false
```

Comment:
Without this, Travis will try to automatically load the currently-private submodule, without an authenticated GitHub user logged in. If I turn off the automatic setup, I can log in simplified-bot and load the submodule myself (lines 18-19).

This is only a temporary measure. It can be changed when the server-core is public.